tests_config/run_spec.js

```diff
@@ -72,22 +75,19 @@ function run_spec(dirname, parsers, options) {
       });
 
       if (AST_COMPARE) {
-        const compareOptions = Object.assign({}, mergedOptions);
-        delete compareOptions.cursorOffset;
-        const astMassaged = parse(input, compareOptions);
-        let ppastMassaged;
-        let pperr = null;
-        try {
-          ppastMassaged = parse(
-            prettyprint(input, path, compareOptions),
-            compareOptions
-          );
-        } catch (e) {
-          pperr = e.stack;
-        }
-
-        test(path + " parse", () => {
-          expect(pperr).toBe(null);
+        test(`${path} parse`, () => {
```

Comment:
Seemed better to stick it inside the test, so that if it throws it's in the context of a test